Testscripts/Linux/VERIFY-LIS-MODULES-VERSION.sh

```diff
@@ -72,22 +73,32 @@ for module in "${HYPERV_MODULES[@]}"; do
     [[ -n $skip ]] || tempList+=("$module")
 done
 HYPERV_MODULES=("${tempList[@]}")
+LogMsg "Target module names: ${HYPERV_MODULES[*]}"
+
+if [ ! $HYPERV_MODULES ]; then
+    LogErr "Target module is empty or null"
+    exit_code=$((exit_code+1))
+fi
```

Comment:
Seems, we have all modules are built-in situation, let me double check.

Reply:
:clock930:

Reply:
if you don't have any other feedback or update with your "double check" result, please close this comment.

Reply:
@juhlee-microsoft In SLES15, all modules are built-in, we can't make it abort it in this situation, also no LIS rpm package installed, we'd better to skip it in this situation. 

Mon Dec 16 00:48:38 2019 : Skipping the built-in modules, hv_storvsc
Mon Dec 16 00:48:38 2019 : Skipping the built-in modules, hv_balloon
Mon Dec 16 00:48:38 2019 : Skipping the built-in modules, hv_vmbus
Mon Dec 16 00:48:38 2019 : Skipping the built-in modules, hid_hyperv
Mon Dec 16 00:48:38 2019 : Skipping the built-in modules, hyperv_keyboard
Mon Dec 16 00:48:38 2019 : Skipping the built-in modules, hv_netvsc
Mon Dec 16 00:48:38 2019 : Skipping the built-in modules, hv_utils

Reply:
@LiliDeng Please clarify your feedback regarding SLES15. Like your saying, all modules in SLES 15 are built-in, and you don't want to get it as 'aborted'. What should we do? If we do not verify any, then what test result should we deliver out?

Reply:
We'd better not to add testskip in line 161, it will miss other failures.

Reply:
Will push a patch to bypass non-CentOS/RHEL distro with TestSkip option in the beginning.